packages/ui-library/src/components/tab-bar/index.ts

```diff
@@ -172,14 +183,24 @@ export class BlrTabBar extends LitElementCustom {
                         role="tab"
                         aria-controls=${`panel-${index}`}
                         class="${navListItemClasses}"
-                        @click=${() => {
+                        @focus=${(event: FocusEvent) => {
+                          if (!isDisabled) {
```

Comment:
Please for consistency purpose move isDisabled inside the respective function and in the template just pass the handler like in our components. i.e. @focus={handleFocus}. Same for blur and click event.

Reply:
We can do that, but it won't match the consistency of other components because `isDisabled` is on a per tab basis instead of it being a component wide variable. We also need to have a check on the `keydown` event to make sure that the 'space' key was pressed. I've moved the disabled check to the function regardless, it might be a bit tidier